layouts/game/list.html

```diff
@@ -1,61 +1,198 @@
-                <th>Compatibility</th>
-                <th>Date Tested</th>
+               <th></th>
+               <th class="sort" data-sort="title">Game Title</th>
+               <th class="sort" data-sort="compatibility">Compatibility</th>
+               <th class="sort" data-sort="date-tested">Date Tested</th>
             </tr>
-        </thead>
-        <tbody class="is-size-7">
+         </thead>
+         <tbody class="list is-size-6">
             {{ range .Pages }}
             {{ $rating := index .Site.Data.compatibility .Params.compatibility }}
             <tr data-key="{{ .Params.section_id }}">
-                <td data-title="{{ .Params.title }}">
-                    <a href="{{ .Permalink }}">{{ .Params.title }}</a>
-                </td>
-                <td data-compatibility="{{ $rating.key }}">
-                    <div class="is-square" style="background-color: {{ $rating.color }}"></div> <span>{{ $rating.name }}</span>
-                </td>
-                <td data-timestamp="{{ .Params.testcase_date }}">
-					{{ if .Params.testcase_date }}
-					<span>{{ dateFormat "January 2, 2006" .Params.testcase_date }}</span>
-					{{ else }}
-					<span></span>
-					{{ end }}
-                </td>
+               <td class="hidden listing-metadata">
+                  <span hidden>{{ .Params.title }} {{ $rating.name }}</span>
+               </td>
+               <td class="game-title" data-title="{{ .Params.title }}">
+                  <a href="{{ .Permalink }}">{{ .Params.title }}</a>
+               </td>
+               <td class="compatibility" data-compatibility="{{ $rating.key }}">
+                  <div class="is-square" style="background-color: {{ $rating.color }}"></div>
+                  <span>{{ $rating.name }}</span>
+               </td>
+               <td class="date-tested" data-timestamp="{{ .Params.testcase_date }}">
+                  {{ if .Params.testcase_date }}
+                  <span>{{ dateFormat "January 2, 2006" .Params.testcase_date }}</span>
+                  {{ else }}
+                  <span></span>
+                  {{ end }}
+               </td>
             </tr>
             {{ end }}
-        </tbody>
-    </table>
+         </tbody>
+      </table>
+      <ul class="pagination paginationBottom is-pulled-left"></ul>
+   </div>
 </div>
 {{ end }}
 
 {{ define "scripts" }}
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.1.1/jquery.min.js" 
+        integrity="sha256-hVVnYaiADRTO2PzUGmuLJr8BLUSjGIZsDYGmIJLv2b8=" crossorigin="anonymous"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/list.js/2.3.1/list.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/3.9.1/chart.min.js"></script>
+
+<script type="text/javascript">
+    $(function() {
+        function getLastSorted(list){
+            var currentSorted = $(list.listContainer).find(".asc, .desc").attr("data-sort");
+            if ('undefined' !== typeof currentSorted)
```

Comment:
```suggestion
            if (!currentSorted)
```
Comparing `undefined` through types are not portable across browsers. Please use standard truthy comparisons.